docs/sdk/dev/development-tools/tooling-config-file.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Tooling config file
-description: The article provides information about a tooling config file that contains settings for supported tools and directives.
+description: This article provides information about a tooling config file that contains settings for supported tools and directives.
```

Comment:
use "document" instead of "article" https://developers.google.com/style/word-list#documentation
```suggestion
description: This document provides information about a tooling config file that contains settings for supported tools and directives.
```